bridge-www/src/pages/SwapWncg.tsx

```diff
@@ -45,52 +52,81 @@ const SwapWncgPage: React.FC<SwapWncgPageProps> = ({ address }) => {
   const [amount, setAmount] = useState<string>("0");
   const burnAmount = useMemo<Decimal | null>(() => {
     try {
-      return new Decimal(amount || "0").mul(new Decimal(10).pow(18))
+      return new Decimal(amount || "0").mul(new Decimal(10).pow(18));
     } catch {
       return null;
     }
   }, [amount]);
 
   return (
-    <Container style={{
-      margin: "30vh 0",
-      minWidth: "100%",
-      overflow: "hidden",
-    }}>
+    <Container
+      style={{
+        margin: "30vh 0",
+        minWidth: "100%",
+        overflow: "hidden",
+      }}
+    >
       <Row justify="center">
-        <Text>Your wNCG : {
-          isLoading
-            ? <Text span={true}>🕑</Text>
-            : <Text weight="bold" span={true}>{
-                  data !== undefined
-                      ? new Decimal(data.value.toString()).div(new Decimal(10).pow(data.decimals)).toString()
-                      : error?.message
-              }</Text>
-        }</Text>
+        <Text>
+          Your wNCG :{" "}
+          {isLoading ? (
+            <Text span={true}>🕑</Text>
+          ) : (
+            <Text weight="bold" span={true}>
+              {data !== undefined
+                ? new Decimal(data.value.toString())
+                    .div(new Decimal(10).pow(data.decimals))
+                    .toString()
+                : error?.message}
+            </Text>
+          )}
+        </Text>
       </Row>
       <Spacer />
       <Spacer />
       <Row justify="center">
-        <TextField label={'To'} onChange={setNcAddress}/>
+        <TextField label={"To"} onChange={setNcAddress} />
       </Row>
       <Spacer />
       <Row justify="center">
-        <TextField label={'Amount'} onChange={setAmount}/>
+        <TextField label={"Amount"} onChange={setAmount} />
       </Row>
       <Spacer />
       <Row justify="center">
-        {
-          contract === null || burnAmount === null || burnAmount.toString().indexOf(".") !== -1 || !isAddress(ncAddress)
-            ? <Text weight={"bold"}>Fill corret values</Text>
-            : <Button onClick={event => {
-              event.preventDefault();    
+        {contract === null ||
+        burnAmount === null ||
+        burnAmount.toString().indexOf(".") !== -1 ||
+        !isAddress(ncAddress) ? (
+          <Text weight={"bold"}>Fill corret values</Text>
+        ) : (
+          <Button
+            onClick={(event) => {
+              if (event.currentTarget === undefined) return;
+              event.preventDefault();
               console.log(contract);
-              contract.burn(BigNumber.from(burnAmount.toString()), ncAddress + "0".repeat(24)).then(console.debug)
-            }}>Burn</Button>
-        }
+
+              const planetId = "0x100000000001";
```

Comment:
please leave comment or add document link about the meaning of `0x100000000001`. because we could have multiple planet ids.

Reply:
Added Comment of planet-ids